fe/fe-core/src/main/java/org/apache/doris/transaction/DatabaseTransactionMgr.java

```diff
@@ -1470,6 +1468,67 @@ private void checkReplicaContinuousVersionSucc(long tabletId, Replica replica, b
         }
     }
 
+    private void checkReplicaContinuousVersionSucc(long tabletId, Replica replica, boolean alterReplicaLoadedTxn,
```

Comment:
add argument:   long  alterWatermarkTxn,    List<Long>  subTxnIds.
And subTxnIds should be the same with replicaPublishTasks